### ISSUE TITLE
Loinc 5787-7:  "Epithelial cells [#/area] in Urine sediment by Microscopy high power field"

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#4823]
Suggest creating a new term for Loinc: 5787-7 [Epithelial cells [#/area] in Urine sediment by Microscopy high power field] 
New term label:Abnormal epithelial cell count in urine
New term comment (if any): Need a parent term to Increased urinary epithelial cell count for LOINC annotation
Your biocurator ID for loinc2hpo (if desired): JGM:lccarmody

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0032211

THe only abnormality is in the increased direction. @kingmanzhang @LCCarmody
